brainstormforce/astra
# AST-738 - Enqueue script missing for astra-admin-settings which throws console error

### Description
- There is an error in the console on Astra options page, we cannot activate any of the Extra free plugins / addon:
- Bulk action trigger
- AST-738

### Screenshots
- https://share.bsf.io/2Nulkm6o

### Types of changes
- Bug fix 

### How has this been tested?
- Check for free plugin install/activate trigger
- Check for any addon activate/deactivate case
- Check for bulk activate/deactivate case
- PS: Make sure to watch console every time
- Please also check notice actions working as we did before
- Task for the same - https://wp-astra.atlassian.net/browse/AST-738
- Check "Get Started" button events working correctly for install/activate Starter Templates plugin

### Checklist:
